src/design-system/asset-input/simple-asset-input.component.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import { Box } from '../box';
+import { Text } from '../text';
+
+import { AmountInput } from './amount-input.component';
+import * as cx from './simple-asset-input.css';
+
+import type { Asset, AssetState } from './asset-input.data';
```

Comment:
Those two types could be moved to the `types.ts` file so simple asset does not import asset's things.

Reply:
@szymonmaslowski Simple asset input is very related to the more complex asset input, so i actually kept the same APIs and data types on purpose. It's also foreseeable that the simple asset input will eventually be extended and slowly merged back with the more complex one, since it was only introduced to ship with less features first.

Reply:
originally i tried to make the asset input more configurable but since its grid layout is not very flexible nor extendable i had to introduce this simpler component for the time being.